decidim-core/app/views/decidim/errors/internal_server_error.html.erb

```diff
@@ -1,10 +1,32 @@
 <div class="wrapper">
   <div class="row">
     <div class="large-8 large-centered">
-    <div class="static__message">
-      <h1 class="heading1 page-title"><%= t(".title") %></h1>
-      <p><%= t(".try_later") %></p>
+      <div class="static__message">
+        <h1 class="heading1 page-title"><%= t(".title") %></h1>
+        <p><%= t(".try_later") %></p>
+        <div class="table-scroll">
+          <table class="table-list">
+          <% @info_hash.map do |key, val| %>
+            <tr>
+              <th><%= t(".#{key}") %></th>
+              <td><%= val %></td>
+            </tr>
+          <% end %>
+          </table>
+        </div>
+        <div class="input-group">
```

Comment:
Regarding the last issue about the "cut" border with the button, it is visually fixed, but:
- The `input-group` wrapper here is unnecessary
- The `button-group` wrapper within the input group

Otherwise it looks now visually good but let's remove the unnecessary wrappers from the DOM.